README.md

```diff
@@ -1,2 +1,70 @@
-# Math-Magicians
-added later
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```

Comment:
- [ ] The README file for this project is not descriptive as mentioned by the previous reviewer. Kindly follow this [template](https://github.com/microverseinc/readme-template) as you've done with previous projects.